mithril.d.ts

```diff
@@ -22,7 +22,7 @@ declare namespace Mithril {
 		*/
 		(
 			selector: string,
-			attributes: Attributes,
+			attributes?: Attributes,
```

Comment:
If it is optional then typing confuses between Attributes and VirtualElement. And think that `m('div', 'text')` its component.

Now need make two variant
```
        (selector: string,
         attributes: Attributes,
         ...children: Array<string | VirtualElement | Component<Controller> | Array<string | VirtualElement | Component<Controller> | Array<string | VirtualElement | Component<Controller>>>>): VirtualElement;
```
and
```
        (selector: string,
         ...children: Array<string | VirtualElement | Component<Controller> | Array<string | VirtualElement | Component<Controller> | Array<string | VirtualElement | Component<Controller>>>>): VirtualElement;
```